mnist/tensorflow/with-model-pool/README.md

```diff
@@ -0,0 +1,21 @@
+# Train and Serve a Model
+
+## Prerequisite
+
+Please make sure both SKAI/dashboard and SKAI/model-pool are installed.
```

Comment:
Pls add install link for dashboard and model-pool.

Reply:
fixed